packages/back-end/test/data-generator/new-generator.ts

```diff
@@ -27,7 +29,7 @@ type ExperimentTableData = TableData & {
   variationId: number;
 };
 type PurchaseTableData = TableData & {
-  amount: number;
+  amount: number | null; // null will get written as \N to file
```

Comment:
@bryce-fitzsimons suggested I use `number | null` instead of overloading a string here. This works fine as we already replace null values when writing CSV, I just had to change it from a blank string to `"\N"` and it all seems to work fine as `\N` is only in purchases.csv in the amount field.

```
% grep -c -r "\\\\N" /tmp/csv  
/tmp/csv/purchases.csv:1125
/tmp/csv/experimentViews.csv:0
/tmp/csv/events.csv:0
/tmp/csv/pageViews.csv:0
/tmp/csv/sessions.csv:0
```